entries/n/netcorecloud.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Netcore Cloud Email API": {
+    "domain": "netcorecloud.com",
+    "additional-domains": [
+      "pepipost.com"
+    ],
+    "tfa": [
+      "email",
+      "totp"
+    ],
+    "documentation": "https://emaildocs.netcorecloud.com/docs/two-factor-authentication-2fa",
+    "keywords": [
+      "communication"
```

Just like in #6017, marketing would be the proper category.
```suggestion
      "marketing"
```

---

I'll change it in that PR instead.